_posts/en/2021-08-20-micro-acquisition-pilot-update2.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: post
+title:  "Micro-Acquisition Pilot: Status Update 2"
+ref: micro-acquisition-update2
+lang: en
+author: "Rachel Muston, in collaboration with the IT Strategy team and Micro-Acquisition Pilot team members"
+date: "2021-08-20"
+excerpt: "The Micro-Acquisition pilot is live! In this blog post we share what we have learned so far and how we are adjusting and improving."
+---
+<!--markdownlint-disable MD033-->
+Since our last status update, the [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html) went live and the [first opportunity](https://micro-acquisition.alpha.canada.ca/2021/06/23/building-custom-Azure-marketplace-policies.html) was posted.
```

Comment:
Should we move links to the footnote as per the guidelines you showed us before to avoid click bait?

I was thinking about moving the links out of the paragraph and put them in a blockquote or as a footnote(1)?

> [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html)

(1): [Micro-Acquisition website](https://micro-acquisition.alpha.canada.ca/en/index.html)

Reply:
Blockquotes require the use of ">" in front of paragraphs.

Footnotes use the following link model `[^1]` in the paragraph and then `[^1]: ` at the bottom of the page.

Reply:
Link to guide: https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#toc11

Reply:
Maybe some excitement could be added, like a "yeah!" at the end of the sentence?

Reply:
Good point @gcharest! I've changed three of the links to footnotes and only kept the links that are related to the changes we talk about in the blog post. What do you think?